starter-core/src/main/java/io/micronaut/starter/feature/acme/Acme.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.acme;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.OneOfFeature;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class Acme implements OneOfFeature {
```

Comment:
This should not implement OneOfFeature. There is only 1 acme feature so there can only be 1. One of is for ensuring only 1 database driver can be included for example.

Reply:
Fixed in 97df988